src/Microsoft.DotNet.Wpf/src/System.Xaml/System/Windows/Markup/ValueSerializer.cs

```diff
@@ -37,10 +23,12 @@ namespace System.Windows.Markup
     /// ValueSerializer with the type to indicate the type converter should be ignored. Implementation of ValueSerializer 
     /// should avoid throwing exceptions. Any exceptions thrown could possibly terminate serialization.
     /// </summary>
-    /// 
     [TypeForwardedFrom("WindowsBase, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35")]
     public abstract class ValueSerializer
     {
+        private static object s_valueSerializersLock = new object();
```

Comment:
Earlier `s_valueSerializersLock` was `readonly`, why are we changing it here ?

Reply:
fixed